test/integration/usageManagementAPI_spec.js

```diff
@@ -162,30 +188,42 @@ var checkUsageNotifications = function (apiKey, value, correlationNumber, callba
 console.log('[LOG]: starting an endpoint for testing...');
 usageAPI_mock.run(test_config.usageAPI_port);
 
-async.each(test_config.databases, function (database, task_callback) {
+after(function (done) {
+    async.eachSeries(test_config.databases, function (database, task_callback) {
+        prepare_test.clearDatabase(database, databaseName, task_callback);
+    }, function (err) {
+        if (err) {
+            return done();
+        } else {
+            return done();
+        }
+    });
+});
```

Comment:
In the done is executed in both the if and the else you can extract it from the if. Additionally, this block is repeated several times in the different tests, maybe it worth having a test utils module for refactoring this kind of methods